src/components/auth/torus/sdk/torus.native.js

```diff
@@ -2,9 +2,12 @@ import DirectNativeSDK from 'torus-direct-react-native-sdk' // eslint-disable-li
 
 class Torus {
   constructor(Config, options) {
+    const { publicUrl } = Config
+
     const redirectUri = 'gooddollar://org.gooddollar/redirect'
+    const browserRedirectUri = `${publicUrl || 'https://dev.gooddollar.org'}/torus/scripts.html`
```

Comment:
```suggestion
    const browserRedirectUri = `${publicUrl}/torus/scripts.html`
```

publicUrl is always set (some default value is added in config.js)

Reply:
Yes, the thing is that the value it defaults to is `window.location.origin`, but that doesn't work on native (leaving it undefined), so it seemed safer this way.
But also, with these changes `REACT_APP_PUBLIC_URL` _should_ always be set in the .envs

Reply:
@serdiukov-o-nordwhale
Should we just assume it's always set and commit your suggestion?

Reply:
@MarianoAmado for native you could set default value in config.js according to the REACT_APP_ENV (e.g. gooddev if env = dev, goodqa if staging, wallet.gooddollar.org if prod)

but anyway applying any default values this should be done in config, not here